### ISSUE TITLE
DynamicSparseArrays.PackedMemoryArray @show method incorrect

### ISSUE THREAD

**Issue by ItamarRocha** [atoptima/DynamicSparseArrays.jl#20]
Reproducing the bug:
```julia
using Coluna

struct FakeModel <: Coluna.ColunaBase.AbstractModel end

model = FakeModel()
Solution = Coluna.ColunaBase.Solution{FakeModel,Int,Float64}

dict_sol = Dict(1 => 2.0, 2 => 3.0, 3 => 4.0)
soldecs = [1,2,3]
solvals = [2.0,3.0,4.0]

primal_sol = Solution(model, soldecs, solvals, 0.0, Coluna.ColunaBase.FEASIBLE_SOL)
```

The show method for primal_sol in the example above give us the following output:

```repl
julia> primal_sol
Coluna.ColunaBase.Solution{FakeModel,Int64,Float64} with 3 entries:
  1 => 2.0
  2 => 3.0
  3 => 4.0
```

While the show method for the primal_sol.sol (type = DynamicSparseArrays.PackedMemoryArray) give us the memory that is not being used too.

```repl
julia> primal_sol.sol
8-element DynamicSparseArrays.PackedMemoryArray{Int64,Float64,DynamicSparseArrays.NoPredictor}:
 2.0
 3.0
 4.0
 0.0
 0.0
 0.0
 0.0
 0.0
 ```

**Comment by guimarqu** [comment]
related to #1

**Comment by guimarqu** [comment]
closed by #39
